src/components/SearchModal/CommonBases.tsx

```diff
@@ -27,11 +27,11 @@ const BaseWrapper = styled.div<{ disable?: boolean }>`
   align-items: center;
   :hover {
     cursor: ${({ disable }) => !disable && 'pointer'};
-    background-color: ${({ theme }) => theme.hoverDefault};
+    background-color: ${({ theme }) => theme.deprecated_hoverDefault};
   }
 
-  color: ${({ theme, disable }) => disable && theme.accentActive};
-  background-color: ${({ theme, disable }) => disable && theme.accentActiveSoft};
+  color: ${({ theme, disable }) => disable && theme.neutral1};
+  background-color: ${({ theme, disable }) => disable && theme.surface3};
```

Comment:
just checking we're deliberately moving away from accent colors here?

Reply:
yep - design team confirmed